xenserver/tests/matchers.py

```diff
@@ -92,31 +75,34 @@ def __init__(self, **fields):
         self.fields = fields
         self.expected_fields = deepcopy(self.default_fields)
         self.expected_fields.update(self.fields)
+        self.matcher = dictmatcher(self.expected_fields)
 
     def match(self, other):
-        return self.expected_fields == other
+        return self.matcher.match(other)
 
-    def __repr__(self):
-        return "<%s %r>" % (type(self).__name__, self.expected_fields)
+    def __str__(self):
+        return "%s(**%r)" % (type(self).__name__, self.expected_fields)
 
 
-class ExpectedXenServerVM(ExpectedXenServerObject):
+class MatchesXenServerVM(BaseXenServerMatcher):
     """
     A matcher for a XenServer VM object.
     """
     default_fields = {
-        'name_label': NO_VALUE,
+        'name_label': Never(),
```

Comment:
"Will never match and always fail, no matter what `x` is. Included for completeness with `Always()`, but if you find a use for this, let us know!" -- https://testtools.readthedocs.io/en/latest/for-test-authors.html#never

I did, and jml was suitably appreciative. ;-)